src/boards/include/boards/pico_w.h

```diff
@@ -99,4 +99,22 @@
 #define CYW43_WL_GPIO_LED_PIN 0
 #endif
 
+// CYW43 GPIO to get VBUS
+#ifndef CYW43_WL_GPIO_VBUS_PIN
+#define CYW43_WL_GPIO_VBUS_PIN 2
+#endif
+
+// VSYS pin is shared with CYW43
+#define CYW43_USES_VSYS_PIN 1
```

Comment:
Apologies if this is a stupid question, but should this also be inside an `#ifndef` ?

Alternatively, given that this is the header-file specifically for Pico W, and there's no way to change which pins the CYW43 is connected to, perhaps _none_ of these settings should be inside an `#ifndef` ? :shrug:

Reply:
I don't know why we use these ifndef in this file - it does seem pointless. Maybe @kilograham knows.

Reply:
For some settings (e.g. the `DEFAULT_UART` stuff) it _does_ make sense to "ifndef it", because  the UART is available on multiple pins, and so the user might choose to have **their** UART wired up to different pins, and they're able to override it by specifying their own values before including the board-header-file. But for other settings (e.g. anything to do with CYW43) I'm not sure if it makes sense to "ifndef it", because there's no way for the user to override the values. All IMHO obviously!

Reply:
yeah we do by convention always `#ifndef` partly just because, but also because a user can take one board definition and make it a basis for something else